python/pyarb.cpp

```diff
@@ -17,21 +23,27 @@ void register_mechanisms(pybind11::module& m);
 void register_morphology(pybind11::module& m);
 void register_profiler(pybind11::module& m);
 void register_recipe(pybind11::module& m);
-void register_sampling(pybind11::module& m);
 void register_schedules(pybind11::module& m);
-void register_simulation(pybind11::module& m);
+void register_simulation(pybind11::module& m, pyarb_global_ptr);
 void register_single_cell(pybind11::module& m);
 void register_spike_handling(pybind11::module& m);
 
 #ifdef ARB_MPI_ENABLED
 void register_mpi(pybind11::module& m);
 #endif
-}
+
+} // namespace pyarb
 
 PYBIND11_MODULE(_arbor, m) {
+    pyarb::pyarb_global_ptr global_ptr(new pyarb::pyarb_global);
+
+    PYBIND11_NUMPY_DTYPE(arb::cell_member_type, gid, index);
```

Comment:
Can we have a quick comment explaining what is going on here?

Reply:
Sure